sdk/tools/gen_baseaddress.py

```diff
@@ -343,6 +344,13 @@ def gen_baseaddress(self, output_file):
         for obj in self.addresses:
             obj.gen_baseaddress(output_file)
 
+def is_gcc():
+    try:
+        result = subprocess.run(['gcc', '--version'], capture_output=True, text=True)
```

Comment:
I assume it works, though it looks a bit run context dependent.
Hopefully, a condition on caller `ntdll_pe` could be found, or something similar.